wiki/Contests/en.md

```diff
@@ -2,291 +2,418 @@
-
-### 2014
-
-| Placing | Results | Original Artists |
-| :-: | :-- | :-- |
-| ![Gold Crown][GCrown] | [yealina](https://osu.ppy.sh/users/139551) - [Parallel Hearts](https://soundcloud.com/yealina/parallel-hearts) | FictionJunction |
-| ![Silver Crown][SCrown] | [Sharlo](https://osu.ppy.sh/users/1622450) - 緋色月下、狂咲ノ絶 -1st Anniversary Remix- | EastNewSound |
-| ![Bronze Crown][BCrown] | [MissTitannia](https://osu.ppy.sh/users/4490361) - *unknown* | *unknown* |
-
-- [Archive sheet for all stages (check by tabs below the spreadsheet)](https://docs.google.com/spreadsheets/d/1XEL-Zb_ldV0l4jpv7IAWzER-LV-uoo9pE2nIb7PI23Y/edit?pli=1#gid=1003675360)
-- [News post](https://osu.ppy.sh/home/news/2014-08-24-osuidol-sign-ups-now-open)
-- [Results post](https://osu.ppy.sh/home/news/2015-01-02-osuidol-2014-results)
-- [Official post](https://osu.ppy.sh/community/forums/topics/234545)
-
-### 2015
-
-| Placing | Results | Original Artists |
-| :-: | :-- | :-- |
-| ![Gold Crown][GCrown] | [Sagi](https://osu.ppy.sh/users/491799) - Systematic Love | [Camelia (Hatsune Miku)](https://www.youtube.com/watch?v=16iOXrf3dQ8) |
-| ![Silver Crown][SCrown] | [CptHampton](https://osu.ppy.sh/users/3200587) - Drumming Song | [Florence + The Machine](https://www.youtube.com/watch?v=TpLXQorSQe8) |
-| ![Bronze Crown][BCrown] | [yealina](https://osu.ppy.sh/users/139551) - Hyadain no Kakakata☆Kataomoi | [Hyadain](https://www.youtube.com/watch?v=WhQvgHM_Nd4) |
-
-- [Archive sheet for all stages (check by tabs below the spreadsheet)](https://docs.google.com/spreadsheets/d/1oqy5nZFQK7Npx2jRVDxaa2zPjDPgtdh5IdWNGrR-qns/edit?pli=1#gid=3048429)
-- [News post](https://osu.ppy.sh/home/news/2015-07-30-osuidol-2015-sign-ups-are-live)
-- [Results post](https://osu.ppy.sh/home/news/2015-12-05-osuidol-2015-results)
-- [Official post](https://osu.ppy.sh/community/forums/topics/352608)
-
-## osu!remix
-
-osu!remix was a contest where musicians were tasked with remixing nekodex's "circles" track, which was osu!'s main menu theme song at the time. It was the first contest to be handled via the [Community Contests listing](https://osu.ppy.sh/community/contests) and has only been run once. Its results can be found [here](https://osu.ppy.sh/community/contests/1).
-
-## Community Contests
-
-Unofficial contests hosted by the communities.
+## Unofficial/community-run contests
+
+*Unofficial contests* are contests that are hosted by members of the community. Contest organizers who are looking to add their contest to the list below are welcome to do so by opening a GitHub pull request in the [osu-wiki repository](https://github.com/ppy/osu-wiki "GitHub") following the [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide).
+
+### osu! Beatmapping World Championship <!-- Yes, OBWC is not an official contest according to https://twitter.com/paichiru/status/1408900730149249025 -->
```

Comment:
```suggestion
### osu! Beatmapping World Championship
```

Reply:
yeah i believe this comment is also supposed to be there to avoid future editors from being confused as to "why is this not listed as official"...